pyquil/api/_base_connection.py

```diff
@@ -52,9 +52,13 @@ def wait_for_job(get_job_fn, ping_time=None, status_time=None):
 
         if status_time and count % int(status_time / ping_time) == 0:
             if job.is_queued():
-                print("job {} is currently queued at position {}".format(job.job_id, job.position_in_queue()))
+                print("job {} is currently queued for running at position {}".format(job.job_id, job.position_in_queue()))
+            if job.is_queued_for_compilation():
```

Comment:
should this be an `elif`?

Reply:
Yup. (No behavioral change, since these conditions are mutually exclusive.)